recipe/meta.yaml

```diff
@@ -14,20 +14,20 @@ source:
         - skip_failing_test.patch  # [osx]
 
 build:
-    number: 3
+    number: 4
     skip: True  # [win]
 
 requirements:
     build:
-        - sqlite
+        - sqlite 3.9*
```

Comment:
The tests `check_virtualtable2` and `check_virtualtable3` failed  with `sqlite 3.11.1` from conda-forge.

Pong @conda-forge/sqlite team.